lib/scala/kg/src/main/scala/models/graphql/AbstractKgGraphQlSchemaDefinition.scala

```diff
@@ -51,22 +51,37 @@ abstract class AbstractKgGraphQlSchemaDefinition extends BaseGraphQlSchemaDefini
     )
   )
 
+  // Input enum types
+  implicit val KgNodeFieldType = deriveEnumType[KgNodeField.Value](
```

Comment:
KgNodeField is probably misnamed. It's not all fields. KgNodeSortableField?

Reply:
Yeah that makes sense, changed to `KgNodeSortableField`